plugins/todo-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,127 @@
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        text:
+                          type: string
+                          description: The contents of the TODO comment
+                        tag:
+                          type: string
+                          description: The tag used, e.g. TODO, FIXME
+                        author:
+                          type: string
+                          description: References author, if any
+                        viewUrl:
+                          type: string
+                          description: URL used to view the file
+                        lineNumber:
+                          type: integer
+                          description: The line number of the file that the TODO occurs at
+                        repoFilePath:
+                          type: string
+                          description: The path of the file containing the TODO within the repo
+                      required:
+                        - text
+                        - tag
+                      additionalProperties: false
+                  totalCount:
+                    type: integer
+                  offset:
+                    type: integer
+                  limit:
+                    type: integer
+                required:
+                  - items
+                  - totalCount
+                  - offset
+                  - limit
+                additionalProperties: false
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: object
+                    properties:
+                      message:
+                        type: string
+      security:
+        - {}
+        - JWT: []
+      parameters:
+        - name: entity
+          in: query
+          required: true
+          schema:
+            type: string
+            description: A reference to the entity to list TODO items for
+        - name: orderBy
+          in: query
+          required: false
+          schema:
+            type: string
+            pattern: '^(text|tag|author|viewUrl|repoFilePath)=(asc|desc)$'
+            description: The field and direction used to sort the listed TODO items
+        - name: filter
+          in: query
+          required: false
+          schema:
+            type: array
+            description: A list of filters used to narrow down the listed TODO items
+            items:
+              type: string
+              pattern: '^(text|tag|author|viewUrl|repoFilePath)=.+$'
+        - name: offset
```

Comment:
It might make sense to add minimum values for these integer fields, something like [this](https://github.com/sennyeya/backstage/blob/306aa032e2ee17a47ea77ca03ff2513cee5d37e8/plugins/catalog-backend/src/schema/openapi.yaml#L80). Not required, but IMO worth describing in the schema.